robolectric/src/test/java/org/robolectric/internal/bytecode/InstrumentingClassLoaderTest.java

```diff
@@ -441,78 +440,91 @@ public void shouldInterceptFilteredStaticMethodInvocations() throws Exception {
 
   @Test
   public void byte_shouldBeHandledAsReturnValueFromInterceptHandler() throws Exception {
+    if (InvokeDynamic.ENABLED) return;
```

Comment:
The problem here is that when using invokedynamic is there is no longer any calls to `ClassHandler.methodInvoked()` which this code is really testing. I think for the foreseeable future we'll continue to run on Travis using both Java 7 and Java 8. Under Java 7 we'll never use invokedynamic anyway so these test will be hit.

As Travis currently runs on JDKu31 we'll not hit any invokedynamic paths yet: https://github.com/travis-ci/travis-ci/issues/3259